migration-toolkit-for-containers/mtc-1-5-release-notes.adoc

```diff
@@ -0,0 +1,35 @@
+[id="mtc-1-5-release-notes"]
+= Migration Toolkit for Containers release notes
+include::modules/common-attributes.adoc[]
+:context: mtc-1-5-release-notes
+
+toc::[]
+
+This document is the release notes for {mtc-full} ({mtc-short}) version {mtc-version}.
+
+The {mtc-full} ({mtc-short}) web console and API, based on Kubernetes custom resources, enable you to migrate stateful application workloads at the granularity of a namespace.
+
+You can migrate from {product-title} 3.7, 3.9, 3.10, or 3.11 to {product-version} and between {product-title} 4 clusters. {mtc-short} enables you to control the migration and to minimize application downtime.
+
+[id="new-features-and-enhancements_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* The *Migration resource* tree on the *Migration details* page of the web console has been enhanced with additional resources, Kubernetes events, and live status information for monitoring and debugging migrations.
+* The web console can contain up to 50 migration plans. Previously, performance was impaired if multiple migration plans were created.
```

Comment:
> The web console can contain up to 50 migration plans.

This isn't correct, unsure where this comes from. There is no concrete limit; it's paginated.

Performance was impaired at *scale*, think 100s of plans. The way this is worded, it suggests a handful of plans impaired the performance, which is also not true.

1.5.0 is designed to support hundreds of plans, which we believe to be a likely quantity based on the default limitation of the amount of namespaces that a plan supports and the fact that many customers will likely have thousands of namespaces.